### PR TITLE
Add Gitea

### PR DESCRIPTION
**Pull request by JonasFranzDEV**
Gitea is popular, community-maintained fork of Gogs.

GitHub: https://github.com/go-gitea/gitea
Website: https://gitea.io
Demo-Website using Gitea: https://try.gitea.io

This Pull-Request adds support for detecting Gitea.

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -3562,6 +3562,23 @@
...
     },
+    "Gitea": {
+      "cats": [
+        47
+      ],
+      "headers": {
+        "Set-Cookie": "i_like_gitea="
+      },
+      "html": [
+        "<div class=\"ui left\">\n\\s+© Gitea Version: [\\d.]+"
```
Comment (jvoisin) [inline src/apps.json]: Please add `\\;version:\\1` so that wappalyzer can get the version

File: src/apps.json [hunk 2]
```diff
@@ -3562,6 +3562,23 @@
...
+      ],
+      "icon": "gitea.svg",
+      "meta": {
+        "keywords": "go,git,self-hosted,gitea"
```
Comment (jvoisin) [inline src/apps.json]: Would it be possible to add `^` and `$` to the regexp?

File: src/apps.json [hunk 3]
```diff
@@ -3562,6 +3562,23 @@
...
+      },
+      "script": "/js/index\\.js\\?v=[a-f0-9]{32}",
```
Comment (jvoisin) [inline src/apps.json]: Please remove this, this is way too false-positive prone.

File: src/apps.json [hunk 4]
```diff
@@ -3562,6 +3562,22 @@
...
+        "Set-Cookie": "^i_like_gitea="
+      },
+      "html": [
+        "<div class=\"ui left\">\n\\s+© Gitea Version: ([\\d.]+)\\;version:\\1"
+      ],
+      "icon": "gitea.svg",
+      "meta": {
+        "keywords": "^go,git,self-hosted,gitea$"
+      },
+      "website": "http://gitea.io"
```
Comment (jvoisin) [inline src/apps.json]: Please use `https` ♥

**Comment by AliasIO** [comment]
Thank you!

**Pull request merged**